tests/test_misc.py

```diff
@@ -43,7 +46,18 @@ def test_multi_dict():
     assert multi_dict.original_items() == [("b", ["abc"])]
 
 
-def test_get_logger(caplog):
+def test_multi_dict_init_empty() -> None:
+    multi_dict = MultiDict(None)
+    assert multi_dict._dict == {}  # pylint: disable=protected-access
+
+
+def test_multi_dict_index_error() -> None:
```

Comment:
Based on the test implementation, I think the name should be improved, something like:
```python
def test__multi_dict__raises_key_error_if_value_of_key_is_empty_list() -> None
```

On the other hand, I have to say that this is a bit counterintuitive and I would treat it as a bug but maybe I am wrong - documentation may help here a little 😉